packages/storage/amplify_storage_s3_dart/example/bin/example.dart

```diff
@@ -179,9 +190,89 @@ Future<void> getUrlOperation() async {
   }
 }
 
+Future<void> copyOperation() async {
+  final sourceKey = prompt('Enter the key of the source object: ');
+  final sourceStorageAccessLevel = promptStorageAccessLevel(
+    'Choose the storage access level associated with the source object: ',
+  );
+  final destinationKey = prompt('Enter the key of the destination object: ');
+  final destinationStorageAccessLevel = promptStorageAccessLevel(
+    'Choose the storage access level associated with the destination object: ',
+  );
+
+  final s3Plugin = Amplify.Storage.getPlugin(AmplifyStorageS3Dart.pluginKey);
+  final copyOperation = s3Plugin.copy(
+    source: S3StorageItemWithAccessLevel(
```

Comment:
These seem really long to have to write multiple times - should we shorten them? What about `S3Item` and `S3ItemWithAccessLevel`?

Reply:
Just generally, maybe changing `S3Storage` prefixes to just `S3`?